### DISCUSSION TITLE
RadioButtonGroup.SelectedValue intended usage

### DISCUSSION THREAD

**Discussion by Cfun1** [xamarin/Xamarin.Forms#12964] [Q&A]
Since I think `RadioButtonGroup.SelectedValue` is not documented yet, if in this xaml https://github.com/xamarin/Xamarin.Forms/blob/2a97a6c5f0e39602cd6c7140c80bc872d0f793e0/Xamarin.Forms.Controls/GalleryPages/RadioButtonGalleries/RadioButtonGroupBindingGallery.xaml#L12 I will hard-code `RadioButtonGroup.SelectedValue="A"` instead, is it supposed to check the correspondent `RadioButton` value or not? because actually it is not. is it the fact that Selection is declared as an object and not a string? https://github.com/xamarin/Xamarin.Forms/blob/2a97a6c5f0e39602cd6c7140c80bc872d0f793e0/Xamarin.Forms.Controls/GalleryPages/RadioButtonGalleries/RadioButtonGroupBindingGallery.xaml.cs#L46 tried to investigate by declaring it as an object `<x:Object..` without success.

In other words, we can set which `RadioButton` of the group is initially checked:

```
<Grid>
...
                <RadioButton Content="Option A" Value="A" Grid.Row="2" IsChecked="True"/>
                <RadioButton Content="Option B" Value="B" Grid.Row="2" Grid.Column="1"/>
                <RadioButton Content="Option C" Value="C" Grid.Row="3"/>
```
Or
```
<Grid RadioButtonGroup.GroupName="{Binding GroupName}" 
                         RadioButtonGroup.SelectedValue="A">
...
                <RadioButton Content="Option A" Value="A" Grid.Row="2" IsChecked="True"/>
                <RadioButton Content="Option B" Value="B" Grid.Row="2" Grid.Column="1"/>
                <RadioButton Content="Option C" Value="C" Grid.Row="3"/>
```
